syntax = "proto2";

package MMPro.micromsg;

import "BaseResponse.proto";

message VerifyUserResponse {
	required BaseResponse _BaseResponse = 1;
	optional string _UserName = 2;
}
